guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Capsule_Server.adoc

```diff
@@ -4,28 +4,35 @@
 The host name of {SmartProxyServer} is referenced by {ProjectServer} components, and all hosts registered to it.
 This procedure ensures that you update all references to the new host name.
 
+[IMPORTANT]
+====
+* Both the `hostname` and `hostname -f` commands must return the FQDN of {SmartProxyServer} or the `{project-change-hostname}` script will fail to complete.
+
+* If the `hostname` command returns the shortname of {SmartProxyServer} instead of the FQDN, use `hostnamectl set-hostname _<old_fqdn>_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
+====
+
 .Prerequisites
+
+* Backup {SmartProxyServer}.
+The `{project-change-hostname}` script makes irreversible changes to {SmartProxyServer}.
+If the renaming process is not successful, you must restore it from a backup.
 +
+{ProjectName} does not provide a native backup method for {SmartProxyServer}.
+For more information, see xref:backing-up-satellite-server-and-capsule-server[].
```

Comment:
I think this shall be shortened to something like "It is advised to perform a backup prior changing a hostname. For more info..."

Reply:
+1

Reply:
"Perform a backup prior to changing a hostname".